Client/Frontend/Browser/TabDisplayManager.swift

```diff
@@ -378,6 +378,33 @@ class TabDisplayManager: NSObject, FeatureFlagsProtocol {
             self.tabDisplayer?.focusSelectedTab()
         }
     }
+    
+    func removeGroupTab(with tab:Tab) {
+        let groupData = indexOfGroupTab(tab: tab)
+        let groupIndexPath = IndexPath(row: 0, section: TabDisplaySection.groupedTabs.rawValue)
```

Comment:
Groups are always at row 0? There can't be more than 1 group?

Reply:
Grid tabs has tableview as a top single cell which has all the groups in it. 
Table contains all groups in it which we do not touch here. 

Grid Collection -> Table View Cell -> [Tab Group Collectionview Cell]